src/apps/lwaftr/lwaftr.lua

```diff
@@ -43,6 +44,43 @@ local function compute_binding_table_by_ipv4(binding_table)
    return ret
 end
 
+local function port_to_psid(port, reserved_ports_bit_count, psid_len)
+   local shift = 16 - (reserved_ports_bit_count + psid_len)
+   return band(rshift(port, shift), (2^psid_len)-1)
```

Comment:
I get what you're saying.  I really appreciate that you think about all these things, but I do want to nudge you (and all of us) towards never making a comment that would leave the patch submitter wondering what to do.  "be careful" is not actionable :)

Let's split the difference and use local INVALID_PSID = 0x10000.  It's out of the uint16 range and in the int32 range which luajit tends to like better.  OK!